File: app/src/main/java/com/fishweeb/practical/PaintColor.java
package com.fishweeb.practical;

import android.graphics.Color;
import android.graphics.Paint;

public class PaintColor
{
    static PaintColor Instance = new PaintColor();

    private Paint Red;
    private Paint Blue;
    private Paint Black;
    private Paint Yellow;

    private PaintColor()
    {
        Red = new Paint();
        Red.setColor(Color.RED);

        Blue = new Paint();
        Blue.setColor(Color.BLUE);

        Black = new Paint();
        Black.setColor(Color.BLACK);

        Yellow = new Paint();
        Yellow.setColor(Color.YELLOW);
    }

    public Paint GetPaint(int _color)
    {
        switch (_color)
        {
            case Color.RED:
                return Red;
            case Color.BLUE:
                return Blue;
            case Color.BLACK:
                return Black;
            case Color.YELLOW:
                return Yellow;
            default:
                return null;
        }
    }
}

File: app/src/main/kotlin/com/fishweeb/practical/PaintColor.kt
package com.fishweeb.practical

import android.graphics.Color
import android.graphics.Paint

class PaintColor
private constructor() {
    private val Red = Paint()
    private val Blue: Paint
    private val Black: Paint
    private val Yellow: Paint

    init {
        Red.color = Color.RED

        Blue = Paint()
        Blue.color = Color.BLUE

        Black = Paint()
        Black.color = Color.BLACK

        Yellow = Paint()
        Yellow.color = Color.YELLOW
    }

    fun GetPaint(_color: Int): Paint? {
        return when (_color) {
            Color.RED -> Red
            Color.BLUE -> Blue
            Color.BLACK -> Black
            Color.YELLOW -> Yellow
            else -> null
        }
    }

    companion object {
        var Instance: PaintColor = PaintColor()
    }
}
